diff --git a/dag2dag/unitex2sxpipe/src/Fst2List.cpp b/dag2dag/unitex2sxpipe/src/Fst2List.cpp
index ba004b1..65baa8e 100644
--- a/dag2dag/unitex2sxpipe/src/Fst2List.cpp
+++ b/dag2dag/unitex2sxpipe/src/Fst2List.cpp
@@ -1538,7 +1538,7 @@ void CFstApp::outWordsOfGraph(int depth)
 			ep = (u_strcmp((unichar *)Eti->contenu,u_epsilon_string)) ? 
 				(unichar *)Eti->contenu : u_null_string;
 
-            if((int)(Eti->transduction)){
+            if(Eti->transduction){
 			tp = (u_strcmp((unichar *)Eti->transduction,u_epsilon_string)) ? 
 				(unichar *)Eti->transduction:u_null_string;
 				} else tp = u_null_string;
diff --git a/lingpipe/lingpipe b/lingpipe/lingpipe
index c51464d..0480229 100755
--- a/lingpipe/lingpipe
+++ b/lingpipe/lingpipe
@@ -195,7 +195,7 @@ Pipelines may be described in configuration files (see lingpipe.conf)
 
 =head1 AUTHORS
 
-Lionel Clément, E<lt>Lionel.Clement@inria.frE<gt>
+Lionel Clement, E<lt>Lionel.Clement@inria.frE<gt>
 
 Eric de la Clergerie, E<lt>Eric.De_La_Clergerie@inria.frE<gt>
 
diff --git a/sxspell/src/sxspeller.c b/sxspell/src/sxspeller.c
index 00f3d4c..cd3b92a 100644
--- a/sxspell/src/sxspeller.c
+++ b/sxspell/src/sxspeller.c
@@ -369,7 +369,7 @@ call_super_light_sxspell (char *mot, SXINT lgth)
   sxspell_init_fsa (mot, lgth, &super_light_FSA);
 
   sxspell_replace (&super_light_FSA, maj2diacritic, T_NB (maj2diacritic), maj2diacritic_weight);
-  sxspell_replace (&super_light_FSA, diacritics, T_NB (diacritics), diacritics_weight); 
+  sxspell_replace (&super_light_FSA, diacritics_and_case, T_NB (diacritics_and_case), diacritics_and_case_weight); 
 
   ret_val = sxspell_do_it (&super_light_FSA);
 
@@ -397,7 +397,7 @@ call_light_sxspell (mot, lgth)
   /* changement de casse */
   sxspell_replace (&light_FSA, min_maj, T_NB (min_maj), min_maj_weight);
   sxspell_replace (&light_FSA, maj_min, T_NB (maj_min), maj_min_weight);
-  sxspell_replace (&light_FSA, diacritics, T_NB (diacritics), diacritics_weight);
+  sxspell_replace (&light_FSA, diacritics_and_case, T_NB (diacritics_and_case), diacritics_and_case_weight);
 
   sxspell_replace_n_p (&light_FSA, twochars_to_diacritics, T_NB (twochars_to_diacritics), twochars_to_diacritics_weight);
   sxspell_replace_n_p (&light_FSA, mm_en_m, T_NB (mm_en_m), mm_en_m_weight);
@@ -470,7 +470,7 @@ call_sxspell (mot, lgth)
   /* changement de casse */
   sxspell_replace (&main_FSA, min_maj, T_NB (min_maj), min_maj_weight);
   sxspell_replace (&main_FSA, maj_min, T_NB (maj_min), maj_min_weight);
-  sxspell_replace (&main_FSA, diacritics, T_NB (diacritics), diacritics_weight);
+  sxspell_replace (&main_FSA, diacritics_and_case, T_NB (diacritics_and_case), diacritics_and_case_weight);
   /* les lettres double'es sont fausses */
   sxspell_replace_n_p (&main_FSA, mm_en_m, T_NB (mm_en_m), mm_en_m_weight);
   /* il faut des consonnes double'es */
diff --git a/sxspell/src/text2dag_semact.c b/sxspell/src/text2dag_semact.c
index 94c7903..36f49e3 100644
--- a/sxspell/src/text2dag_semact.c
+++ b/sxspell/src/text2dag_semact.c
@@ -829,7 +829,7 @@ call_light_sxspell (dico, FSA_ptr, mot, lgth, f, is_deterministic)
   /* changement de casse */
   sxspell_replace (FSA_ptr, min_maj, T_NB (min_maj), min_maj_weight /* change_case_weight */);
   sxspell_replace (FSA_ptr, maj_min, T_NB (maj_min), maj_min_weight);
-  sxspell_replace (FSA_ptr, diacritics, T_NB (diacritics), diacritics_weight);
+  sxspell_replace (FSA_ptr, diacritics_and_case, T_NB (diacritics_and_case), diacritics_and_case_weight);
 
   sxspell_replace_n_p (FSA_ptr, twochars_to_diacritics, T_NB (twochars_to_diacritics), twochars_to_diacritics_weight);
   sxspell_replace_n_p (FSA_ptr, mm_en_m, T_NB (mm_en_m), mm_en_m_weight);
@@ -1010,7 +1010,7 @@ call_super_light_sxspell (dico, FSA_ptr, mot, lgth, f)
   sxspell_init_fsa (mot, lgth, FSA_ptr);
 
   sxspell_replace (FSA_ptr, maj_min, T_NB (maj_min), 1 /* maj_min_weight */); // et non -1
-  sxspell_replace (FSA_ptr, diacritics, T_NB (diacritics), 1 /* diacritics_weight */); // et non -1
+  sxspell_replace (FSA_ptr, diacritics_and_case, T_NB (diacritics_and_case), 1 /* diacritics_and_case_weight */); // et non -1
 
   ret_val = 0;
 
@@ -1067,7 +1067,7 @@ call_sxspell (dico, FSA_ptr, mot, lgth, f, is_deterministic)
   /* changement de casse */
   sxspell_replace (FSA_ptr, min_maj, T_NB (min_maj), min_maj_weight);
   sxspell_replace (FSA_ptr, maj_min, T_NB (maj_min), maj_min_weight);
-  sxspell_replace (FSA_ptr, diacritics, T_NB (diacritics), diacritics_weight);
+  sxspell_replace (FSA_ptr, diacritics_and_case, T_NB (diacritics_and_case), diacritics_and_case_weight);
   /* les lettres double'es sont fausses */
   sxspell_replace_n_p (FSA_ptr, mm_en_m, T_NB (mm_en_m), mm_en_m_weight);
   /* il faut des consonnes double'es */
